Sources/PackageManagerDocs/Documentation.docc/IntroducingPackages.md

```diff
@@ -4,8 +4,49 @@ Learn to create and use a Swift package.
 
 ## Overview
 
-overview content here....
+A package consists of a `Package.swift` manifest file along with source files, resources, and other assets. 
+The manifest file, or package manifest, defines the package's name and its contents using the PackageDescription module.
 
-### First Section
+Each package declares `Products`, a list of what the package produces.
+Types of products include libraries, executables, and plugins:
 
-First section content
+- A library defines a module that can be imported by other code. 
+- An executable is a program that can be run by the operating system.
+- A plugin is executable code that the Swift Package Manager may use to provide additional commands or build capabilities.
+
+The package can also declare `Dependencies`, most frequently other Swift packages that provide modules you use.
+Dependencies can also provide macros, plugins, or reference system or binary (non-source) dependencies.
+
+Each product is made up of one or more `Targets`, the basic building block of a Swift package.
+Each target specifies an output, may declare one or more dependencies on other targets within the same package and on products vended by the package’s dependencies.
```

Comment:
What does "output" refer to here?

Reply:
@neonichu Good question - I think it's "module", but I wasn't 100% confident on the term that's used for what it provides, and how multiple targets can be referenced when you're defining a `library` product. If I switch that out to `module`, I think that works.

Please let me know what a better word would be here if it isn't `module`.